### ISSUE TITLE
Unable to resolve actions due to name change

### ISSUE THREAD

**Issue by mike-xu-homex** [google-github-actions/release-please-action#421]
### TL;DR

When we trigger the github action, we see the following error:
`Unable to resolve actions. Repository not found : googlecloudplatform/release-please-action.`

### Expected behavior

GitHub should be able to resolve this action without changing the `-uses:` parameter.

### Observed behavior

GitHub action reports the error above if we set
`-uses: GoogleCloudPlatform/release-please-action@v2`
or
`-uses: GoogleCloudPlatform/release-please-action@v3`

Only `-uses: google-github-actions/release-please-action@v2` or `-uses: google-github-actions/release-please-action@v3` works for now.

### Action YAML

```yaml
on:
  push:
    branches:
      - master
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - id: release
        uses: GoogleCloudPlatform/release-please-action@v2
        with:
          release-type: node
          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Chores","hidden":false},{"type":"revert","section":"Reverts","hidden":false}]'
```

**Comment by ChristianTellefsen** [comment]
The readme still says to use GoogleCloudPlatform/release-please-action. 

What should we do - was this an intended change, and we should now change all our actions in all our projects, or is this a mistake that you are going to revert?

**Comment by bcoe** [comment]
@ChristianTellefsen the correct canonical organization for the action is `google-github-actions` now, the README should be updated.

**Comment by dbsanfte** [comment]
Try not to break these things please :(
